spec/models/field_of_operation_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe FieldOfOperation do
+  describe "#field_of_operation" do
```

Comment:
I think we can update this to be `fatality_notices` to match the name of the method being tested.